website/docs/pushdown.md

```diff
@@ -19,9 +19,9 @@ You can learn more about query pushdown in our [blog post](https://blog.ferretdb
 
 ## Supported types and operators
 
-The following table shows all operators and types that FerretDB pushdowns on PostgreSQL backend.
+The following table shows all operators and types that FerretDB uses pushdown on PostgreSQL backend.
```

Comment:
Something is not right there gramatically

Reply:
Oops, now rearranged a bit, see what you think.